src/main/java/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package strategy.impl;
```

Comment:
The package declaration should not be followed by an empty line. Consider removing the line to adhere to the checklist.